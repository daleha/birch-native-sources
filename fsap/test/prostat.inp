sample.pro
n
temp.prostat.out
