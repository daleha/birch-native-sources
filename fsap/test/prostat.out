PROSTAT      Version  8/30/93
pI39                             74 aa
Molecular weight:    8245.0

Nonpolar side chains:
Gly(G)              2 (0.027)
Ala(A)              7 (0.095)
Val(V)              5 (0.068)
Leu(L)              9 (0.122)
Ile(I)              2 (0.027)
Met(M)              1 (0.014)
Phe(F)              4 (0.054)
Pro(P)              0 (0.000)
Neutral polar side chains:
Ser(S)              4 (0.054)
Thr(T)              6 (0.081)
Cys(C)              8 (0.108)
Asn(N)              4 (0.054)
Gln(Q)              2 (0.027)
Tyr(Y)              1 (0.014)
Trp(W)              1 (0.014)
Charged polar side chains:
Asp(D)              4 (0.054)
Glu(E)              4 (0.054)
His(H)              4 (0.054)
Lys(K)              5 (0.068)
Arg(R)              1 (0.014)
Other:
Asx(B)              0 (0.000)
Glx(Z)              0 (0.000)
Unk(X)              0 (0.000)
